src/services/findAllReferences.ts

```diff
@@ -526,7 +526,7 @@ namespace ts.FindAllReferences {
     function getTextSpan(node: Node, sourceFile: SourceFile, endNode?: Node): TextSpan {
         let start = node.getStart(sourceFile);
         let end = (endNode || node).getEnd();
-        if (isStringLiteralLike(node)) {
+        if (isStringLiteralLike(node) && (end - start) > 2) {

```

Comment:
Unterminated string at EOF?

Reply:
[Response here](https://github.com/microsoft/TypeScript/pull/47535#discussion_r789178876)

But generally, sort of, yeah - short ill-formed strings without quotes.